src/commands/ptzplayaudio.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * @type {Record<string, number>}
+ */
+const audioNameToIdMap = {
+  alarm: 37,
+  siren: 49,
+  emergency: 40,
+  trespassing: 43,
+  camera: 33,
+  hello: 1,
+  despacito: 0,
+  ringtone: 35,
+  dog: 44
+}
+
+const aliases = [
+  'ptzplayaudio',
+  'playclip',
+  'playaudio'
+]
+
+/**
+ * @type {Array<import('./types.d.ts').Command>}
+ */
+const commands = aliases.map(alias => ({
```

Comment:
I think the command structure should have an aliases prop (during the command loading you can generate an obj/map to do quick lookups) instead of us needing to create multiple identical objects for each alias (and having a single obj per command means we could more easily use the data to power the site commands list etc.)